cmd/copy.go

```diff
@@ -16,6 +16,10 @@ import (
 var getSessionIDRegex = regexp.MustCompile(`^sess:([^/]+)`)
 
 func Copy(cmd *cobra.Command, args []string) error {
+	if len(args) < 2 {
+		return fmt.Errorf("At least two path arguments are required for the Copy command")
+	}
+
```

Comment:
You don't need this. Cobra validates this for you [here](https://github.com/unweave/cli/blob/master/main.go#L163).

Reply:
This is handled in the new branch